ovos_utils/log.py

```diff
@@ -166,8 +184,17 @@ def init_service_logger(service_name):
     except ImportError:
         LOG.warning("ovos_config not available. Falling back to defaults")
         _cfg = dict()
-    _log_level = _cfg.get("log_level", "INFO")
+    # First try the logging section
+    log_config = _cfg.get("logging")
+    # get the "logs" from the root level if there is one
     _logs_conf = _cfg.get("logs") or {}
+    if log_config:
+        # if "logs" is defined in "logging" use that
+        _logs_conf = log_config.get("logs") or _logs_conf
+        _cfg = log_config.get(service_name) or log_config
+        # if "logs" is redefined in "logging.<service_name>" use that
+        _logs_conf = _cfg.get("logs") or _logs_conf
```

Comment:
this could use a bit more explanation in comments for the next dev. took me a minute and i'm pretty sure i wrote the previous version, so it'll be tougher on somebody else. logic is sound, just a bunch of the same variables in a small space

Reply:
yes, apologies - I was going for a minimal change. I'll explain better in some comments if you're happy with the logic

Reply:
On a second view, I do spot one possible improvement:
```suggestion
        _logs_conf = {} // falsey
        if log_config:
            _logs_conf = log_config.get("logs") // successful or falsey
        _logs_conf = _logs_conf or _cfg.get("logs") //successful or problem
        if not _logs_conf:
            raise someError
        _logs_conf["level"] = _log_level
```

That may not be strictly perfect, as I wrote it in this input box on the fly, so be very very careful before you decide to use the "accept suggestion" button rather than coding

Reply:
as I read your suggestion the test on #197 is already true (#191)
Also we don't need to raise an Error as it defaults to an empty dict in #190 and then we default _log_level to INFO... so... ??
I'm closing this PR and moving this to a new one so I'll add some comments and hopefully we can improve it in that PR :)